websitetoolbox.com.apexforum.json

```diff
@@ -0,0 +1,36 @@
+{
+  "providerId":"WebsiteToolbox.com",
+  "providerName":"Website Toolbox",
+  "serviceId":"apexforum",
+  "serviceName":"Website Toolbox Forum",
+  "version": 1,
+  "syncPubKeyDomain" : "websitetoolbox.com",
+  "logoUrl":"https://cdn.websitetoolbox.com/images/logos/wt_logo_transparent.png",
+  "description":"Enables the root of a domain to work with Website Toolbox.",
+  "variableDescription":"",
+  "records":[
+    {
+      "type":"REDIR301",
```

Comment:
Not all DNS Providers support the REDIR command; so be warned that this template may not work well with all the DNS Providers.

Reply:
Ok. Our load balancers currently have ever-changing IP addresses, but I'm looking into having at least one static IP address that can be used here instead.